alphagov/paas-cf
# [#113262177] Lint checks for duplicate YAML keys

## What

Enable lint checks to find duplicate keys in static and dynamically generated YAML documents. Please see the individual commit messages for more information.

## How to review

1. Check whether Travis is passing
1. Install the dependencies and run locally (see README and would welcome feedback on how easy this is)
1. Try adding a duplicate key, run the tests again, and see if they fail

## Who can review

Not @dcarley
